tests/test_parse_fasta.py

```diff
@@ -1,11 +1,13 @@
-from parse_fasta import parse_fasta, discard_ambiguous_seqs, nucleotide_frequencies, map_reads
+from pathlib import Path
+
+from src.parse_fasta import parse_fasta, discard_ambiguous_seqs, nucleotide_frequencies, map_reads
```

Comment:
This works now because we have installed `src` and even though `src/` is not in _this_ folder, it's now in `PYTHONPATH` and so we can import it from anywhere as long as we are in the current Conda environment